docs/kb/api-version-3/prospects.md

```diff
@@ -19,7 +19,7 @@ Search criteria may be used together in any combination and/or order unless othe
 | `assigned_to_user`     | `integer/string` | `<any_positive_integer>, <any_string>`                                 | Selects prospects based on whether they are assigned to a specified user. Users can be specified by their email address or their Pardot IDs.Note: Using `assigned_to_user` overrides the `assigned` criteria.                                                                                                                                                                                                         |
 | `created_after`        | `string`         | `today, yesterday, last_7_days, this_month, last_month, <custom_time>` | Selects prospects that were created after the specified time. If a `<custom_time>` is used, ensure that the specified date is formatted using GNU Date Input Syntax.                                                                                                                                                                                                                                                |
 | `created_before`       | `string`         | `today, yesterday, last_7_days, this_month, last_month, <custom_time>` | Selects prospects that were created before the specified time. If a `<custom_time>` is used, ensure that the specified date is formatted using GNU Date Input Syntax.                                                                                                                                                                                                                                               |
-| `deleted`              | `boolean`        | `true, false`                                                          | Selects prospects based on whether they have been deleted.Default value: false.                                                                                                                                                                                                                                                                                                                                  |
+| `deleted`              | `string`         | `true, false, all`                                                     | Selects prospects based on whether they have been deleted. Default value: `false`.                                                                                                                                                                                                                                                                                                                                |
```

Comment:
We should have PAN review to make sure there is not additional info needed for "all"

Reply:
I think having an explanation of why we have 3 possible values for a boolean. True/false are self explanatory. But "all" needs an extra comment.